ui/app/AppLayouts/stores/RootStore.qml

```diff
@@ -54,6 +54,7 @@ QtObject {
     property var accounts: walletSectionSendInst.accounts
     // Not Refactored Yet
 //    property var profileModelInst: profileModel
+    property var tokensModelWallet: walletSectionAllTokens.getTokenList()
```

Comment:
this needs to be checked/tested

Reply:
I've tested with an account that has some `ETH` in testnet and it doesn't validates as expected:

https://github.com/status-im/status-desktop/assets/97019400/e752f146-b97c-4b34-bd6f-2cdc0ed6d0c6